integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -34,6 +71,9 @@ describe(`[gatsby-source-wordpress] Build default options`, () => {
       console.log(`Waiting for WPGraphQL to be ready...`)
       await urling({ url: `http://localhost:8001/graphql`, retry: 100 })
       console.log(`WPGraphQL is ready`)
+      console.log(`Waiting for plugins to be ready...`)
+      await pluginsAreReady()
+      console.log(`Plugins are ready`)
```

Comment:
locally the tests randomly fail if not all wpgql plugins are activated yet. This doesn't happen on ci because ci is so much slower than a local machine. This was annoying enough that I added this